routes/index.js

```diff
@@ -0,0 +1,22 @@
+// 1. IMPORTACIONES
+const express 		= require("express")
+const router		= express.Router()
+
+
+
+// 2. RUTEO
+router.get("/home", (req, res) => {
```

Comment:
Excelente Alan aca yo dejaria esta ruta como la raiz del proyecto recuerda que es la ruta que se ejecuta cuando alguien va a la raiz del proyecto si le dejamos home nos lanza el cannot get por que no hay nada que mostrar en la raiz 👀 
```suggestion
router.get("/", (req, res) => {
```